docs/developer-guide/nodejs-api.md

```diff
@@ -130,13 +130,18 @@ The return value is an object containing the results of the linting operation. H
                     column: 23,
                     message: "Expected a semicolon."
                 }
-            ]
+            ],
+            errorCount: 1,
+            warningCount: 0
         }
-    ]
+    ],
+    errorCount: 1,
+    warningCount: 0
 }
 ```
 
-The top-level report object has a `results` array containing all linting results for files that had warnings or errors (any files that did not product a warning or error are omitted). Each file result includes the `filePath` and a `messages` array. The `messages` array contains the result of calling `linter.verify()` on the given file.
+The top-level report object has a `results` array containing all linting results for files that had warnings or errors (any files that did not product a warning or error are omitted). Each file result includes the `filePath`, a `messages` array, `errorCount` and `warningCount`. The `messages` array contains the result of calling `linter.verify()` on the given file. The `errorCount` and `warningCount` gives the exact number of errors and warning respectively on the given file.
+The top-level report object also has `errorCount` and `warningCount` which gives the exact number of errors and warning respectively on all the files.
```

Comment:
The Markdown parser on the site has been known to be finicky. This line is formatted like a new paragraph, but it lacks a preceding empty line, so I'm not sure whether it will choke. Can you either add a blank line before this if it's supposed to be a separate paragraph, or make sure it's part of line 143?